src/javascript/app/pages/resources/trading_times/trading_times.ui.js

```diff
@@ -30,13 +30,28 @@ const TradingTimesUI = (() => {
         }
 
         const date = moment.utc();
-        $date.attr('data-value', toISOFormat(date));
+        const isoFormattedDate = toISOFormat(date);
+        $date.attr('data-value', isoFormattedDate);
         DatePicker.init({
             selector: '#trading-date',
             minDate : 0,
             maxDate : 364,
         });
         $date.val(localize('Today'));
+        // Create a label to be friendlier
+        const $label = $('label[for=trading-date]');
+        $('<span/>')
+            .class('ux-date foot-note')
+            .appendTo($label);
```

Comment:
why not something like `$label.append($('<span/>', { class: 'ux-date foot-note' });`?